tests/api/test_controller_opdsfeed.py

```diff
@@ -148,7 +148,9 @@ def test_feed(
                 last_item.sort_author,
                 last_item.id,
             ]
-            expect = "key=%s" % url_quote_plus(json.dumps(expected_pagination_key))
+            expect = "key=%s" % quote_plus(
+                json.dumps(expected_pagination_key), safe=","
```

Comment:
`url_quote_plus` does not escape `,` while `quote_plus` does. So I add the `safe=","` parameter so that `,` are not escaped. This does not change the CM behaviour, just makes sure that the test creates the correct expected key.